app/modules/sightings/models.py

```diff
@@ -729,24 +789,39 @@ def send_identification(
             log.warning('send_identification called without id_configs')
             self.stage = SightingStage.failed
             return
-
-        log.debug(
-            f'In send_identification for cnf:{config_id}  algo:{algorithm_id} '
-            f'Ann UUID:{annotation_uuid}  Ann content_guid:{annotation_sage_uuid}'
-        )
-
         # Message construction has to be in the task as the jobId must be unique
         job_uuid = uuid.uuid4()
+        job_uuid_str = str(job_uuid)
+
+        algorithm = self._get_algorithm_name(config_id, algorithm_id)
+        debug_context = f'Sighting:{self.guid}, Annot:{annotation_uuid}, Ann content_guid:{annotation_sage_uuid} algorithm:{algorithm}, job:{job_uuid}'
+        num_jobs = len(self.jobs)
+        log.debug(f'{debug_context}, In send_identification, num jobs {num_jobs}')
+
         matching_set_query = matching_set_query or self.id_configs[config_id].get(
             'matching_set'
         )
         algorithm = self.id_configs[config_id]['algorithms'][algorithm_id]
+        self.jobs[job_uuid_str] = {
+            'matching_set': matching_set_query,
+            'algorithm': algorithm,
+            'annotation': str(annotation_uuid),
+            'annotation_sage_uuid': str(annotation_sage_uuid)
+            if annotation_sage_uuid
+            else 'None',
+        }
+
         if not annotation_sage_uuid:
             log.warning(
-                f'Sage Identification on Sighting({self.guid}) Job({job_uuid}) '
-                f'aborted due to no content_guid on Annotation {annotation_uuid}'
+                f'{debug_context}, Sage Identification aborted due to no content_guid on Annotation '
             )
-            self.stage = SightingStage.failed
+            self.jobs[job_uuid_str]['active'] = False
+            self.jobs[job_uuid_str]['success'] = False
+            self.jobs[job_uuid_str]['failure_reason'] = 'No content guid on Annotation'
```

Comment:
@naknomum, this is what we were talking about, the job itself now has a failure reason